sdk/spring/azure-spring-boot-test-selenium-common/src/main/java/com/azure/test/aad/common/SeleniumITHelper.java

```diff
@@ -48,4 +71,24 @@ public void destroy() {
         driver.quit();
         app.close();
     }
+
+    private void setPathExecutableRecursively(File file) {
+        if (!file.exists()) {
```

Comment:
Can we delete this check?

Reply:
in this project, yes , we can delete it, but I recommend to reserve it.